tutorials/plot_background_filtering.py

```diff
@@ -894,6 +894,103 @@ def baseline_plot(x):
 #     (see also :func:`mne.filter.construct_iir_filter` for options).
 #     For more inforamtion, see e.g. `FieldTrip band-pass documentation`_.
 #
+# Reporting Filters
+# =================
+# On page 45 in Widmann *et al.* [7]_, there is a convenient list of important
+# filter parameters that should be reported with each publication:
+#
+# 1. filtertype (high-pass, low-pass, band-pass, band-stop, FIR, IIR)
+# 2. cutoff frequency (including definition)
+# 3. filter order (or length)
+# 4. roll-off or transition bandwidth
+# 5. passband ripple and stopband attenuation
+# 6. filter delay (zero-phase, linear-phase, non-linear phase) and causality
+# 7. direction of computation (one-pass forward/reverse,or two-pass forward and
+#    reverse)
+#
+# In the following, we will address how to deal with these parameters in MNE:
+#
+#
+# Filter type
+# -----------
+# Depending on the function or method used, the filter type can be specified.
+# To name an example. in :func:`mne.filter.create_filter`, the relevant
+# arguments would be `l_freq`, `h_freg`, `method`, and if the method is fir:
+# `fir_window`, and `fir_design`.
+#
+#
+# Cutoff frequency
+# ----------------
+# The cutoff of fir filters in MNE is defined as half-amplitude cutoff in the
+# middle of the transition band. That is, if you construct a lowpass fir filter
+# with `h_freq = 40.`, the filter function will provide a transition bandwidth.
+# The desired half-amplitude cutoff of the lowpass fir filter is then at:
+# `h_freq + transition_bandwidth/2.`
```

Comment:
double-backticks on code.

You can leave single ones on things that should in principle link (such as var names) if you want. We haven't converged on those, though most of these instances in the codebase are still double-backticks.